### PR TITLE
feat: 485 drag and drop

### PR DESCRIPTION
**Pull request by mateuszszczecina**
## Jira issue(s)
https://selleolabs.atlassian.net/browse/LC-485
https://selleolabs.atlassian.net/browse/LC-491

## Overview

Add drag and drop for questions and options. 
Change options property name (position -> display order)
Add select to fill in the blank to choose the type (dnd, text) 
Change pricing component to be compatible with figma

### CODE REVIEW

File: apps/web/app/modules/Admin/EditCourse/CourseLessons/NewLesson/QuizLessonForm/components/QuestionWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <Accordion.Root type="single" collapsible>
+      <Accordion.Item value={`item-${questionIndex}`}>
+        <div
+          className={`border p-2 mt-4 rounded-xl transition-all duration-300 ${
```
Comment (typeWolffo) [inline apps/web/app/modules/Admin/EditCourse/CourseLessons/NewLesson/QuizLessonForm/components/QuestionWrapper.tsx:33]: cn() :D

File: apps/web/app/modules/Admin/EditCourse/CourseLessons/NewLesson/QuizLessonForm/components/TrueOrFalseQuestion.tsx [hunk 2]
```diff
@@ -84,106 +77,107 @@ const TrueOrFalseQuestion = ({ form, questionIndex }: TrueOrFalseQuestionProps)
...
-                                className="text-error-500 bg-error-50 ml-3 cursor-pointer w-7 h-7 group-hover:text-white group-hover:bg-error-600 rounded-lg p-1"
-                                onClick={handleRemoveQuestion}
-                              />
-                            </div>
-                          </TooltipTrigger>
-                          <TooltipContent
-                            side="top"
-                            align="center"
-                            className="bg-black ml-4 text-white text-sm rounded shadow-md"
+        <div className={"p-2 rounded-xl border-0 transition-all duration-300"}>
```
Comment (typeWolffo) [inline apps/web/app/modules/Admin/EditCourse/CourseLessons/NewLesson/QuizLessonForm/components/TrueOrFalseQuestion.tsx:80]: redundant curly

File: apps/api/src/seed/nice-data-seeds.ts
Comment (wielopolski) [inline apps/api/src/seed/nice-data-seeds.ts:1]: We want to have both type in seed - FillInTheBlanksDnd and FillInTheBlanksText, now we have only FillInTheBlanksDnd

**Review by wielopolski — COMMENTED** [review]
LGTM 🍻

File: apps/api/src/seed/nice-data-seeds.ts
Comment (mateuszszczecina) [reply apps/api/src/seed/nice-data-seeds.ts:1]: Sure i will add one more

**Review by typeWolffo — CHANGES_REQUESTED** [review]

File: apps/web/app/components/SortableList/SortableList.tsx [hunk 3]
```diff
@@ -36,9 +37,18 @@ export function SortableList<T extends BaseItem>({
...
-}: SortableListProps<T>) {
+  isQuiz = false,
+}: SortableListProps<T> & { isQuiz?: boolean }) {
   const [active, setActive] = useState<Active | null>(null);
-  const activeItem = useMemo(() => items.find((item) => item.id === active?.id), [active, items]);
+
+  const activeItem = useMemo(() => {
+    if (isQuiz && active) {
+      const activeOrder = Number(active.id);
+      return items.find((item) => (item.displayOrder as number) === activeOrder);
```
[the remark this answers is not in the document]
Comment (mateuszszczecina) [reply apps/web/app/components/SortableList/SortableList.tsx:47]: You are right

**Review by typeWolffo — APPROVED** [review]
:shipit:

**Pull request merged**